charts/hlf-k8s/templates/deployment-config-operator.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: apps/v1
```

Comment:
Is this operator required for the orderer ? If it's not the case maybe we should add a condition that prevent from deploying it in the orderer cluster this way we do not consume resources we do not need.

Reply:
Fixed: https://github.com/SubstraFoundation/hlf-k8s/pull/45/commits/d30ec8434d620c767a034ef8863dc841ff71e1aa